openedx/core/djangoapps/cors_csrf/tests/test_middleware.py

```diff
@@ -63,6 +64,7 @@ def cb_check_req_is_secure_false(request, callback, args, kwargs):
         self.assertIs(res, SENTINEL)
         self.assertTrue(request.is_secure())
 
+    @skip('Edraak: Not valid anymore')
```

Comment:
I doubt this is correct. We should double check after undoing the CSRF changes above.

Reply:
Done